less/blog-page.less

```diff
@@ -0,0 +1,82 @@
+.blog {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+
```

Comment:
Great job with your css here. Good use of flexbox properties, good use of industry-standard class names. Nice responsive and user-friendly design. Overall, excellent work.